### PR TITLE
bpo-38321: Fix _testcapimodule.c warning

### PR DESCRIPTION
**Pull request by vstinner**
Fix the following warning:

    modules\_testcapimodule.c(6409):
    warning C4146: unary minus operator applied to unsigned type,
    result still unsigned

https://bugs.python.org/issue38321

### CODE REVIEW

**Review by eduardo-elizondo — APPROVED** [review]
Thanks! LGTM

File: Modules/_testcapimodule.c [hunk 1]
```diff
@@ -1952,7 +1952,7 @@ unicode_asutf8andsize(PyObject *self, PyObject *args)
         return NULL;
     }
 
-    buffer = PyUnicode_AsUTF8AndSize(unicode, &utf8_len);
```
Comment (eduardo-elizondo) [inline Modules/_testcapimodule.c:1955]: Unrelated change?

**Review by eduardo-elizondo — APPROVED** [review]

File: Modules/_testcapimodule.c (hunk 1 above)
Comment (vstinner) [reply Modules/_testcapimodule.c:1955]: Yeah, my editor removes trailing spaces. Trailing spaces must die!
